.github/workflows/python_publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
```

Comment:
Are you not supposed to do: `pip install -r requirements-dev.txt` and add `build` to `requirements-dev.txt`?

Reply:
I don't think we want to ship dev dependencies ( e.g. pytest ) so no. Up to you though.

When I downloaded the artifact after my testing it looked fine and was able to install it. https://github.com/corytodd/jsondiff/actions/runs/9228722397

Reply:
I do see now that only `build` is required here as the tests run in a different workflow.

However, adding `build` to `requirements-dev.txt` wouldn't ship it though, would it? It only considers `requirements-dev.txt` if you install it with `pip install .[dev]` according to `pyproject.toml`? But if you ever wanted to invoke the build command locally, you'd already have it installed?

Reply:
Oh good point yeah, having build locally is useful. I can add that in probably tomorrow since I'm traveling today. You might be able to push that change to my branch as repo owner if you need that today.

Reply:
I've added build to the dev dependencies. Let me know if that's sufficient.